docs/src/development/troubleshoot.md

```diff
@@ -96,6 +96,15 @@ platform project:clear-build-cache -p <PROJECT_ID>
 
 The next build for each environment is likely to take longer as the cache rebuilds.
 
+### Access the project
+
+In most cases, issues accessing a project is caused by missing permissions for a given user.
+For more information see [User administration](../administration/users.md).
```

Comment:
As I've mentioned, I'd prefer links that aren't the name of the article despite Google's recommendations. I'll formalize it into the style guide soon, hopefully next week. For now, see [gov.uk's guidelines](https://readabilityguidelines.co.uk/content-design/links/#8-use-sentence-case). Also applies below.

Reply:
This also seems unresolved?